.storybook/config.ts

```diff
@@ -1,13 +1,37 @@
 import { addParameters, configure } from '@storybook/html';
 import { create } from '@storybook/theming';
 
+// List of categories to sort the stories in the order of the InstantSearch.js
+// API reference.
+// See https://www.algolia.com/doc/api-reference/widgets/js/
+const categories = [
+  'Basics',
+  'Metadata',
+  'Pagination',
+  'Refinements',
+  'Results',
+  'Sorting',
+];
+
 addParameters({
   options: {
     theme: create({
       base: 'light',
       brandTitle: 'InstantSearch.js',
       brandUrl: 'https://github.com/algolia/instantsearch.js',
     }),
+    storySort(a: any[], b: any[]) {
+      const categoryA = a[1].kind.split('|')[0];
+      const categoryB = b[1].kind.split('|')[0];
+
+      if (categories.indexOf(categoryA) === categories.indexOf(categoryB)) {
+        return 0;
+      }
+
+      return categories.indexOf(categoryA) < categories.indexOf(categoryB)
```

Comment:
```suggestion
      return categories.indexOf(categoryA) - categories.indexOf(categoryB)
```

this is the same no?

Reply:
(or the other order, i'm always confused about that haha)